aiida_mlip/data/model.py

```diff
@@ -36,17 +35,17 @@ class ModelData(SinglefileData):
         Calculate the hash of a file.
     _check_existing_file(file: Union[str, Path]) -> Path:
         Check if a file already exists and return the path of the existing file.
-    __init__(file: Union[str, Path], architecture: str, \
+    __init__(file: Union[str, Path], architecture: str, 
         filename: Optional[str] = None, **kwargs: Any) -> None:
         Initialize the ModelData object.
-    set_file(file: Union[str, Path], filename: Optional[str] = None, \
+    set_file(file: Union[str, Path], filename: Optional[str] = None, 
         architecture: Optional[str] = None, **kwargs: Any) -> None:
         Set the file for the node.
-    local_file(file: Union[str, Path], architecture: str, \
+    local_file(file: Union[str, Path], architecture: str, 
         filename: Optional[str] = None) -> 'ModelData':
         Create a ModelData instance from a local file.
-    download(url: str, architecture: str, filename: Optional[str] = None, \
-        cache_dir: Optional[Union[str, Path]] = None, \
+    download(url: str, architecture: str, filename: Optional[str] = None, 
+        cache_dir: Optional[Union[str, Path]] = None, 
             force_download: Optional[bool] = False) -> 'ModelData':
         Download a file from a URL and save it as ModelData.
```

Comment:
It won't let me suggest this, but I think you can replace all of this with

```
    Methods
    -------
    set_file(file, filename=None, architecture=None, **kwargs)
        Set the file for the node.
    local_file(file, architecture, filename=None):
        Create a ModelData instance from a local file.
    download(url, architecture, filename=None, cache_dir=None, force_download=False)
        Download a file from a URL and save it as ModelData.
```
In general you don't need to do the type hints/non-public functions for this section: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring